### PR TITLE
Catalog documentation cleanup

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**
- clean up of the documentation related to catalog endpoints
- add information about disabled catalogs

### CODE REVIEW

**Review by eoliveauakeneo — APPROVED** [review]

File: content/swagger/parameters.yaml [hunk 1]
```diff
@@ -121,18 +121,35 @@ with_quality_scores__product_models:
...
+  required: false
+  x-from-version: "SaaS"
+updated_after:
+  name: updated_after
+  in: query
+  description: Filter products that have been updated AFTER the provided date (Only available on Catalogs endpoints)
+  type: string
+  format: date
+  required: false
+  x-from-version: "SaaS"
```
Comment (RiffVentura) [inline content/swagger/parameters.yaml:155]: missing blank line

File: content/swagger/resources/app_catalogs/routes/app_catalogs_id_products.yaml [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                    }
+                ]
+            }
+        }
+    401:
+      $ref: "#/responses/401Error"
+    403:
+      $ref: "#/responses/403CatalogError"
+    404:
+      $ref: "#/responses/404CatalogError"
```
Comment (RiffVentura) [inline content/swagger/resources/app_catalogs/routes/app_catalogs_id_products.yaml:215]: missing empty line

File: content/swagger/resources/app_catalogs/routes/app_catalogs_id_products_uuid.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                }
+              ]
+            }
+          }
+    401:
+      $ref: "#/responses/401Error"
+    403:
+      $ref: "#/responses/403CatalogError"
+    404:
+      $ref: "#/responses/404CatalogError"
```
Comment (RiffVentura) [inline content/swagger/resources/app_catalogs/routes/app_catalogs_id_products_uuid.yaml:50]: missing empty line

File: content/swagger/responses.yaml [hunk 4]
```diff
@@ -144,3 +144,21 @@ DeprecatedAssetReferenceFileUploaded:
...
+  }
+404CatalogError:
+  description: Catalog not found
+  x-details: The catalog id given in the URI does not exist
+  schema:
+    $ref: "#/definitions/Error"
+  examples: {
+    "code": 404,
+    "message": "Catalog \"65f5a521-e65c-4d7b-8be8-1f267fa2729c\" does not exist or you can't access it."
+  }
```
Comment (RiffVentura) [inline content/swagger/responses.yaml:164]: missing empty line

**Review by RiffVentura — APPROVED** [review]

**Pull request merged**